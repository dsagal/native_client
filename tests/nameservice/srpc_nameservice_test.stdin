rpc manifest_test * s("")
