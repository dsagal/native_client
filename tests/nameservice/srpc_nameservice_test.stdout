rpc call initiated manifest_test::s
rpc call complete manifest_test::s
output 0:  s("ManifestTest: basic connectivity ok\n")

